packetbeat/_meta/config/beat.reference.yml.tmpl

```diff
@@ -13,6 +13,16 @@
 # keyword to sniff on all connected interfaces.
 packetbeat.interfaces.device: {{ call .device .GOOS }}
 
+# The network CIDR blocks that are considered "internal" networks for
+# the purpose of network perimeter boundary classification.
+packetbeat.interfaces.home_networks:
```

Comment:
How hard would it be to reuse the named ranges from the `when.network` condition in processors? https://www.elastic.co/guide/en/beats/filebeat/current/defining-processors.html#condition-network

Reply:
Just to clarify for this one. You're asking about/thinking to support the same names that the processor does in this option (i.e. `loopback`, `unicast`, `multicast`, `interface_local_multicast`, `link_local_unicast`, `link_local_multicast`, `private`) and not talking about trying to change the logic to internally use processors correct? Supporting those special names as shorthand for CIDRs seems pretty straightforward.

Reply:
Correct, just offering a way to use those named ranges here for convenience.

Reply:
Since network.direction is internal/external, should we maybe call these "internal_networks" instead of "home_networks"?  It also looks like we use "internal" through most of the code.

Reply:
@leehinman I'd be fine with that name change, it's more consistent.